src/OpenTelemetry.Instrumentation.Http/README.md

```diff
@@ -62,9 +65,42 @@ public class Program
 }
 ```
 
+#### For Metrics
+
+The following example demonstrates adding HTTP client instrumentation with the
+extension method `.AddHttpClientInstrumentation()` to a console application.
+
+The metric and its attributes being implemented are following Opentelemetry
+[metrics semantic
+conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#metric-httpclientduration).
+
+| Name  | Instrument Type | Unit | Description | Attributes |
+|-------|-----------------|------|-------------|------------|
+| `http.client.duration` | Histogram | `ms` | Measures the duration of outbound HTTP requests. | http.method, http.scheme, http.status_code, http.flavor, net.peer.name |
```

Comment:
Can add a link to metric here inside the table https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#metric-httpclientduration

Reply:
Along with this comment: https://github.com/open-telemetry/opentelemetry-dotnet/pull/4216#discussion_r1115147196
Do you mean that we can instead update the Attributes column in this table to be a link to the spec?

I am thinking maybe documenting what's currently being implemented in the package may be a helpful info for the users.
As some of the attributes are optional and we don't currently implement all of the attributes listed in the spec.

Reply:
I meant to remove the attributes column entirely. And update the `Name` column to [http.client.duration](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/http-metrics.md#metric-httpclientduration)

> I am thinking maybe documenting what's currently being implemented in the package may be a helpful info for the users.
As some of the attributes are optional and we don't currently implement all of the attributes listed in the spec.

That is a good point, we have the same thing on traces side. I think we should be consistent, we can revisit it adding it later when the conventions are stable. Would like to hear feedback from others as well..

Reply:
removed the attributes column for now for consistency